frame/include/bli_config_macro_defs.h

```diff
@@ -152,10 +152,10 @@
 // leading dimensions (ie: column strides) within the BLAS compatibility layer.
 // A value of 32 results in the compatibility layer using 32-bit signed integers
 // while 64 results in 64-bit integers. Any other value results in use of the
-// C99 type "long int". Note that this ONLY affects integers used within the
+// C99 type "size_t". Note that this ONLY affects integers used within the
```

Comment:
This is discussing the BLAS layer so I'm pretty sure you mean `int`, not `size_t`?